core/network/Cargo.toml

```diff
@@ -12,25 +12,42 @@ readme = "README.md"
 
 [dependencies]
 avalanche-types = { path = "../../crates/avalanche-types", features = ["message"] }
+byteorder = "1.4.3"
 cert-manager = "0.0.10" # https://github.com/gyuho/cert-manager
 log = "0.4.20"
-rustls = { version = "0.21.5", features = ["logging", "dangerous_configuration"]} # https://github.com/rustls/rustls/tags
+rustls = { version = "0.21.5", features = ["logging", "dangerous_configuration"] } # https://github.com/rustls/rustls/tags
 rcgen = "0.10.0"
 hyper-rustls = "0.24.1"
 rustls-native-certs = "0.6.3"
+probabilistic-collections = { version = "0.7.0", features = ["serde"] }
 hyper = { version = "0.14.27", features = ["full"], optional = true }
 tokio-rustls = { version = "0.24.1", optional = true }
+tokio = { version = "1.32.0", features = ["full"] }
```

Comment:
Adding the `full` tokio set of features is really big. Can we only enable the features we definitely need?